extension.ts

```diff
@@ -151,7 +151,7 @@ export async function activate(context: vscode.ExtensionContext) {
 
     // Delete modehandler once all tabs of this document have been closed
     for (let editorIdentity of ModeHandlerMap.getKeys()) {
-      let modeHandler = await ModeHandlerMap.get(editorIdentity);
+      const modeHandler = ModeHandlerMap.get(editorIdentity);
```

Comment:
We need `await` here. The resolved value is consumed immediately after.

Reply:
ModeHandlerMap.get() is not even async. Only getOrCreate() is async